cluster/environments/azure-multiple-clusters-waf-tm-apimgt/aks-centralus.tf

```diff
@@ -0,0 +1,89 @@
+resource "azurerm_resource_group" "centralrg" {
+  name     = "${var.central_resource_group_name}"
+  location = "${var.central_resource_group_location}"
+}
+
+# local variable with cluster and location specific
+locals {
+  central_rg_name                 = "${azurerm_resource_group.centralrg.name}"
+  central_rg_location             = "${azurerm_resource_group.centralrg.location}"
+  central_prefix                  = "${local.central_rg_location}-${var.cluster_name}"
+  central_flux_clone_dir          = "${local.central_prefix}-flux"
+  central_kubeconfig_filename     = "${local.central_prefix}_kube_config"
+  central_ip_address_out_filename = "${local.central_prefix}_ip_address"
+}
+
+# Creates vnet
+module "central_vnet" {
+  source = "../../azure/vnet"
+
+  resource_group_name     = "${local.central_rg_name }"
+  resource_group_location = "${local.central_rg_location}"
+  subnet_names            = ["${var.cluster_name}-aks-subnet"]
+  address_space           = "${var.central_address_space}"
+  subnet_prefixes         = "${var.central_subnet_prefixes}"
+  tags = {
+    environment = "azure-multiple-clusters"
+  }
+}
+
+# Creates aks cluster
+module "central_aks" {
+  source = "../../azure/aks"
```

Comment:
Migrate this portion to use the aks-gitops module over a remote git source (see azure-single-keyvault for example).

Reply:
This terraform template is from the existing api-mgmt branch. I will check with Nate on what needs to be done here.